eva-accession-clustering-automation/cluster_one_assembly.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2020 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import sys
+import datetime
+import logging
+from create_clustering_properties import create_properties_file
+from create_clustering_properties import check_valid_sources
+from create_clustering_properties import check_vcf_source_requirements
+from config_custom import get_args_from_private_config_file
+from ebi_eva_common_pyutils.command_utils import run_command_with_output
+
+logger = logging.getLogger(__name__)
+timestamp = datetime.datetime.now().strftime("%Y%m%d%H%M%S")
+
+
+def generate_bsub_command(assembly_accession, properties_path, clustering_artifact, automation_timestamp):
+    job_name = 'cluster_' + assembly_accession
+    log_file = assembly_accession + '_cluster_' + timestamp + '.log'
+    error_file = assembly_accession + '_cluster_' + timestamp + '.err'
+
+    command = 'bsub -J ' + job_name + ' -o ' + log_file + ' -e ' + error_file + ' -M 8192 -R "rusage[mem=8192]" ' + \
```

Comment:
Sure we want to hard code the memory requirement here?... 8GB default looks ok but do we foresee a scenario where we might need more (or even less)?

Reply:
Good idea, I added a parameter but left the 8G as default